sdk/storage/Azure.Storage.Common/tests/Azure.Storage.Common.Tests.csproj

```diff
@@ -8,6 +8,14 @@
     <IsPackable>false</IsPackable>
     <RootNamespace>Azure.Storage.Tests</RootNamespace>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Should we eventually move these tests to the Aes Gcm Tests package to avoid having to do this in the Common Tests package?